Makefile

```diff
@@ -24,6 +24,8 @@ DOCKER_BUILD_ARGS := $(if $(DOCKER_VERSION), "--build-arg=DOCKER_VERSION=$(DOCKE
 DOCKER_RUN_OPTS := $(TRAEFIK_ENVS) $(TRAEFIK_MOUNT) "$(TRAEFIK_DEV_IMAGE)"
 DOCKER_RUN_TRAEFIK := docker run $(INTEGRATION_OPTS) -it $(DOCKER_RUN_OPTS)
 DOCKER_RUN_TRAEFIK_NOTTY := docker run $(INTEGRATION_OPTS) -i $(DOCKER_RUN_OPTS)
+DOCKER_BUILD := docker build
```

Comment:
I understood from the PR description that you want to customize the build parameters, but here you're customizing the build command.

I see how that could be used to customized both, but wouldn't it possibly be better to just specify the parameters?

Either way, wouldn't you also want to inject your customizations through, say, an environment variable, and read that in the Makefile? Right now, it looks like you'd have to modify your local copy of the Makefile to achieve what you want.